uperf-wrapper/redis_utils.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python
```

Comment:
In my opinion, this code shouldn't be tied to uperf-wrapper, should be somewhere else like the utils module

Reply:
yes, Initially i did put it in utils but was having issues importing the file. I also created a __init__.py there as you can see in the commit.  I will check it out again. Thanks for the feedback

Reply:
We're now using python3, so imports must be absolute. i.e
`from utils.redis.utils import RedisUtils`